File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n, i, j;
    do
    {
        printf("Height: ");
        n = get_int();
    }
    while (n > 23 || n < 0);
    for (i = 0; i < n; i++)
    {
        for (j = 1; j < n - i; j++)
        {
            printf(" ");
        }
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

n = -1
while n < 0 or n > 23:
    print("Height: ", end="")
    n = cs50.get_int()

for i in range(n) :
    for j in range(n - i - 1) :
        print(" ", end="")
    for j in range(i + 2) :
        print("#", end="")
    print("")